### ISSUE TITLE
💅 useImportType rule with style: inlineType doesn't convert separated type imports

### ISSUE THREAD

**Issue by PaulRBerg** [biomejs/biome#8450]
### Environment information

```bash
Biome v2.3.8
macOS Tahoe 26.1
```

### Rule name

style/useImportType

### Expected result

When `style` is set to `inlineType`, separated type imports like:

```ts
import type { Indexed } from "./types";
```

should be converted to inline type imports:

```ts
import { type Indexed } from "./types";
```

**Actual behavior:** The code stays as `import type { Indexed }` and is not transformed.

### Additional context

This was discussed in [#6026](https://github.com/biomejs/biome/discussions/6026#discussioncomment-13850264).

The reverse direction works correctly: `separatedType` converts `import { type Indexed }` to `import type { Indexed }`. However, `inlineType` does not perform the opposite transformation.

Without this fix, users cannot use the documented workaround to merge `import type` and regular `import` statements via organize imports, since inline types are required for merging to work.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Again, ask you for the biome rage output and a reproduction

**Comment by PaulRBerg** [comment]
Biome v2.3.9 works correctly now.

Whatever bug I was experiencing before (when I was using v2.0 beta) is gone!

**Comment by Conaclos** [comment]
This was fixed in https://github.com/biomejs/biome/pull/7316
